website/content/docs/configuration/index.mdx

```diff
@@ -87,6 +87,13 @@ path to the file.
 
 - [`plugins`](/boundary/docs/configuration/plugins): Configures options for plugins.
 
+## Signals
+On `SIGHUP`, worker and controller processes will reload their configuration file to pick up updated values
```

Comment:
I'd add a caveat that not all configuration file values are able to be updated at runtime. Bonus points if you figure out which ones and ensure they're documented as such on the appropriate config pages :-)

Reply:
Thanks! I'll hunt these down